util/topgen/templates/toplevel_racl_pkg.sv.tpl

```diff
@@ -37,7 +37,7 @@ package top_${topcfg["name"]}_racl_pkg;
         "range_mapping"    : mapping.get('range_mapping', []),
         "policy_names"     : policy_names,
         "racl_group"       : racl_group,
-        "module_name"      : m['name'],
+        "m"                : m,
```

Comment:
nit: perhaps `module` would be more descriptive than `m` for the key. `m` is perfectly cromulent for the variable holding values from an iterator, but it's less readable if it's used as API across files.